### PR TITLE
More correct monster names

### PR DESCRIPTION
**Pull request by Hypexion**
Part two of my crusade to make names more consistent. This PR covers Fegundo, Seep, Rock Golems and Rock Giants.

This doesn't cover any of the snakes, or engineers, because they're all internally consistent. Also, the snake file names make it pretty obvious what lives inside them.

### CODE REVIEW

**Comment by mrimer** [comment]
@Hypexion , would you please rebase this PR?

File: DRODLib/RockGiant.cpp [hunk 1]
```diff
@@ -20,11 +20,15 @@
  * Portions created by the Initial Developer are Copyright (C) 1995, 1996, 
  * 1997, 2000, 2001, 2002, 2005 Caravel Software. All Rights Reserved.
  *
- * Contributor(s): Mike Rimer (mrimer)
+ * Contributor(s):
  *
  * ***** END LICENSE BLOCK ***** */
 
+//Splitter.cpp
```
Comment (mrimer) [inline DRODLib/RockGiant.cpp:27]: Remove this comment.

File: DRODLib/RockGiant.h [hunk 2]
```diff
@@ -20,33 +20,43 @@
...
  * 1997, 2000, 2001, 2002, 2005 Caravel Software. All Rights Reserved.
  *
- * Contributor(s): Mike Rimer (mrimer)
+ * Contributor(s):
  *
  * ***** END LICENSE BLOCK ***** */
 
-#ifndef ROCKGOLEM_H
-#define ROCKGOLEM_H
+//Splitter.h
```
Comment (mrimer) [inline DRODLib/RockGiant.h:27]: Clean up this comment.

**Review by mrimer — CHANGES_REQUESTED** [review]
Looks good!
Thank you for the good neighborly effort to keep the codebase clean.
I suggested minor tweaks.

The PR says that Texts/Speech.uni has a conflict.  Would you please resolve this, and then I'll merge.  (Sorry, I don't know of an easy way to do that.  Do you have a utility that can perform a readable diff on the .uni files?)

**Comment by mrimer** [comment]
Oh, and may I request to update the DRODLib.2019.vcxproj project file (etc) with the new file names?

**Comment by Hypexion** [comment]
@mrimer I've updated the VS2019 project file, fixed up the comments and resolved the conflict.

It appears that most Git clients don't recognise the UCS2 encoding the uni files use as text, which is certainly a decision they are allowed to make. Luckily, TortioseGit does treat them as text, allowing you merge and diff them normally. Changing the encoding might help, but since a free tool exists that works, it's probably safer to leave it.

**Review by mrimer — APPROVED** [review]

**Pull request merged**